networking/src/Ntp/Client.hs

```diff
@@ -172,7 +172,7 @@ sendLoop cli addrs = do
     let poll        = ntpPollDelay (ncSettings cli)
 
     -- send packets and wait until end of poll delay
-    sock <- atomically $ readTVar $ ncSockets cli
+    sock <- readTVarIO $ ncSockets cli
```

Comment:
Any reason for the change to `readTVarIO`?

Reply:
Just `stylish-haskell` thing.

Reply:
Huh, interesting. AFAIK it doesn't make any difference. stylish-haskell tends to annoy me far more than it pleases me.